src/result.js

```diff
@@ -113,10 +115,20 @@ class Result {
 }
 
 function insertIntoIndexHTML(html, head, body) {
-  html = html.replace("<!-- EMBER_CLI_FASTBOOT_BODY -->", body);
+  const headComment = "<!-- EMBER_CLI_FASTBOOT_HEAD -->";
+  const bodyComment = "<!-- EMBER_CLI_FASTBOOT_BODY -->";
+
+  const warnMissing = function (missing) {
+    chalk.red(`Fastboot was not able to find ${missing} in base HTML`);
```

Comment:
I think we should throw an error as well. We don't expect developers to use `fastboot` library and call `result.html()` without including `ember-cli-fastboot` addon. They go hand in glove.